routes/api/contacts.js

```diff
@@ -1,25 +1,118 @@
-const express = require('express')
+const express = require('express');
+const {
+  listContacts,
+  getContactById,
+  removeContact,
+  addContact,
+  updateContact,
+  updateStatusContact,
+} = require('../../controllers/contacts');
 
-const router = express.Router()
+const validate = require('../api/validation');
+
+const router = express.Router();
 
 router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+  try {
+    const contacts = await listContacts();
+    res
+      .status(200)
+      .json({ status: 'success', code: 200, data: { contacts } });
+  } catch (error) {
+    next(error);
+  }
+});
 
 router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+  try {
+    const { contactId } = req.params;
+    const contact = await getContactById(contactId);
+    if (contact) {
+      return res
+        .status(200)
+        .json({ status: 'success', code: 200, data: { contact } });
+    } else {
+      return res
+        .status(404)
+        .json({ status: 'error', code: 404, message: 'Not found' });
+    }
```

Comment:
```suggestion
    if (contact) 
      return res
        .status(200)
        .json({ status: 'success', code: 200, data: { contact } });

      return res
        .status(404)
        .json({ status: 'error', code: 404, message: 'Not found' });
```
Dzięki temu że używasz return nie potrzeba if else tutaj, tylko if. Ma sens? :)

Reply:
Jeśli jest kontakt drugie return się nie wykona bo pierwsze return już zwróci res.

Reply:
Taki trick upraszcza kod i jego czytanie.